src/fs/rock/RockSwapDir.cc

```diff
@@ -1121,6 +1109,7 @@ Rock::SwapDir::hasReadableEntry(const StoreEntry &e) const
 
 namespace Rock
 {
+
```

Comment:
Please remove this out-of-scope whitespace change.

Reply:
Done.